hydra_agent/querying_mechanism.py

```diff
@@ -672,22 +672,20 @@ def main():
     Take URL as an input and make graph using initilize function.
     :return: call query function for more query.
     """
-    url = input("url>>>").strip()
+    url = input("url>>>").strip().rstrip('/')
     if url == "exit":
         print("exit...")
```

Comment:
No `print` statements are allowed in code. Code shall use Python standard library's `logging` package:
```
import logging

logger = logging.getLogger(__module__)
logger.info('This is a warning')
```
Logger can be initialised in every module or in the `__init__` of the package

Reply:
Okay, will look out for that. Already replaced all the ones I had in my code, these are prints from the previous implementation on the nlp part. Will make sure to not use in all the new modules and I think I'll leave the refactoring of the nlp part when we get there.